.gitignore

```diff
@@ -94,7 +94,7 @@ game_shader_dx9_srv.so
 game_shader_dx9_srv.so.dbg
 game_shader_dx9.dylib
 game_shader_dx9.dylib.dYSM/
-mp/game/bin/
+mp/game/bin/*.dll
```

Comment:
Linux generates a `GameUI.so` and `GameUI.so.dbg` file here, and maybe a few more, I forget.

You'll want to use the [gitignore whitelist](https://git-scm.com/docs/gitignore#_pattern_format) format here and just whitelist the base.fgd file in addition to the ignoring of the directory:

```
mp/game/bin/
!mp/game/bin/base.fgd
```